clang/lib/3C/DeclRewriter.cpp

```diff
@@ -337,6 +379,110 @@ void DeclRewriter::rewriteMultiDecl(DeclReplacement *N, RSet &ToRewrite) {
     VisitedMultiDeclMembers.insert(TN);
 }
 
+void DeclRewriter::rewriteNamedInlineStruct(DeclReplacement *N, RSet &ToRewrite, std::vector<Decl *> SameLineDecls) {
```

Comment:
This method seems too similar to the original multi-declaration rewriting method. I feel like there must be a way to use the same method for both.

Reply:
Yeah for sure there is, one of my earliest drafts actually had them together. (I think the idea is you add an additional parameter and have it be added in explicitly at call time). At any rate, I decided not to on the offchance that it will cause the already well-functioning rewriteMultiDecl to break for some reason. (But I can unify the two methods in my next commit).